src/view/form-edit-view.js

```diff
@@ -135,8 +138,16 @@ export default class FormEditView extends AbstractStatefulView {
     this.element.querySelector('.event__type-group').addEventListener('change', this.#typeClickHandler);
     this.element.querySelector('.event__input--price').addEventListener('change', this.#priceChangeHandler);
     this.element.querySelector('.event__input--destination').addEventListener('change', this.#destinationChangeHandler);
+    const availableOffers = this.element.querySelector('.event__available-offers') || '';
+    if(availableOffers === '') {
+      return;
+    }
+    availableOffers.addEventListener('change', this.#offerChangeHandler);
```

Comment:
Блоки if отделяем от другого кода 
```suggestion
    const availableOffers = this.element.querySelector('.event__available-offers') || '';
    
    if(availableOffers === '') {
      return;
    }
    
    availableOffers.addEventListener('change', this.#offerChangeHandler);
```